chapters/vulkan_profiles.adoc

```diff
@@ -0,0 +1,521 @@
+* The LunarG Desktop Baseline 2022 profile for desktop platforms requires Vulkan 1.1 with specific extensions
+* Future profiles may target newer Vulkan versions as they become more widely supported
+
+This approach allows developers to target specific capability sets without being tied to a particular Vulkan version, providing more flexibility in application development.
+
+== Official Profiles
+
+The Khronos Group maintains several profiles, which are defined in the link:https://github.com/KhronosGroup/Vulkan-Profiles/tree/main/profiles[Vulkan-Profiles repository]. The only official profiles are the Khronos Roadmap profiles and the Android Baseline profile. The others are maintained by Google and LunarG:
+
+* *LunarG Desktop Baseline 2022* - A baseline profile for desktop platforms
+(link:https://github.com/KhronosGroup/Vulkan-Profiles/blob/main/profiles/VP_LUNARG_desktop_baseline_2022[JSON Definitions])
+* *Android Baseline 2022* - A baseline profile specifically for Android devices (link:https://vulkan.lunarg.com/doc/sdk/latest/windows/profiles_api_library.html[Documentation], link:https://github.com/KhronosGroup/Vulkan-Profiles/blob/main/profiles/VP_ANDROID_baseline_2022.json[JSON Definition])
+* *Roadmap 2024* - A forward-looking profile that hardware vendors are
+encouraged to support in future implementations (link:https://vulkan.lunarg.com/doc/sdk/latest/windows/profiles_api_library.html[Documentation],
+link:https://docs.vulkan.org/spec/latest/appendices/roadmap.html#roadmap-2024[JSON Definition], link:https://docs.vulkan.org/spec/latest/appendices/roadmap.html[roadmap])
+
+These profiles are versioned by year to allow for evolution over time while maintaining backward compatibility. You can find sample code demonstrating the use of these profiles in the link:https://github.com/KhronosGroup/Vulkan-Samples/tree/main/samples/tooling/profiles[Vulkan-Samples repository].
+
+=== LunarG Desktop Baseline 2022 Profile Details
+
+The LunarG Desktop Baseline 2022 profile (`VP_LUNARG_DESKTOP_BASELINE_2022`) is designed for desktop platforms. It's described as "A profile generated by the intersection of a collection of GPUInfo.org device reports to support a large number of actual systems in the Vulkan ecosystem. This profile is meant to be a usage example for Vulkan application developers."
+
+The real purpose of the LunarG Desktop Baseline profiles is to provide examples to Vulkan application developers to create profiles for their applications. It's important to note that these profiles have some limitations - for example, Windows ARM / Qualcomm GPUs are not supported because these GPUs were not released or just released when the profiles were generated.
+
+If LunarG were to create a Desktop Baseline 2025 in the future, it would likely include Windows ARM / Qualcomm GPUs, which would make the Desktop Baseline 2025 both a subset and superset (likely Vulkan 1.3) of the Desktop Baseline 2024.
```

Comment:
Ahah, I wasn't expecting you would take literally my explaination in the documentation, but that's fine!

Reply:
I thought your explanation was perfect, why try to improve upon perfection ;-)